xclim/indices/run_length.py

```diff
@@ -58,7 +58,7 @@ def use_ufunc(
         If ufunc_1dim is "auto", returns True if the array is on dask or too large.
         Otherwise, returns ufunc_1dim.
     """
-    if ufunc_1dim is False and freq is not None:
+    if ufunc_1dim is True and freq is not None:
```

Comment:
Just a heads-up here, but evaluating against `True` here will result in `False` if `ufunc_1dim` is a valid string. Otherwise, evaluating `ufunc_1dim` for "truthiness" (ie: ` if ufunc_1dim and freq is not None`) will accept `True` or a valid string.

Reply:
Thanks, that does what I want then. I want a warning only if `ufunc_1dim` was asked specifically, i.e. set to `True` as opposed to `"auto"` or `"from_context"`. I want the user to be notified this is not a valid choice.